lib/components/RecentBlocks.js

```diff
@@ -52,11 +57,31 @@ class RecentBlocks extends React.Component {
         block.coinbase = Array.isArray(block.coinbase)
           ? block.coinbase.join('')
           : block.coinbase;
+
+        // convert unix timestamp to human readable string
+        block.time = Number.isInteger(block.time)
+          ? new Date(block.time * 1000)
+              .toISOString()
+              .replace(/T/, ' ')
+              .replace(/\..+/, '')
+          : block.time;
+      });
+
+      if (SPV)
+        this.colHeaders = this.colHeaders.filter(
+          header => !this.SPVomit.includes(header)
+        );
+
+      if (clientChain === 'bitcoincash')
+        this.colHeaders = swap(this.colHeaders, this.bcashSwap);
```

Comment:
Doesn't need to be in this, but makes me think that to add extra flexibility for future additional chain support that this could almost be something like `weightSupport` or `segwitSupport` somewhere in categorizations of supported chains. On the other hand, maybe this is better too since who knows where else cash and bitcoin could diverge in the future.